BDArmory/ModuleWeapon.cs

```diff
@@ -1671,11 +1636,116 @@ void Aim()
                 {
                     turret.smoothRotation = false;
                 }
-                turret.AimToTarget(finalTarget);
+                turret.AimToTarget(
+                    FiringSolutionVector != null 
+                    ? part.transform.position + (Vector3)FiringSolutionVector * 1099511627776 // 2^40
+                    // because I cannot figure out why the AimToTarget method aims sideways without the large number multiplier
+                    : finalAimTarget);
```

Comment:
A rather inelegant way to use a method which wants a target location when all we have is a direction vector.